tests/sles4sap/redirection_tests/hanasr_schedule_tests.pm

```diff
@@ -65,7 +65,7 @@ sub run {
     my %scenarios;
     my @failover_actions = split(",", get_var("HANASR_FAILOVER_SCENARIOS", 'stop,kill'));
     for my $method (@failover_actions) {
-        my $test_name = ucfirst($method) . "_DB-$primary_site-$primary_db";
+        my $test_name = ucfirst($method) . "_primary_DB-$primary_site";
```

Comment:
is this one for name used on the openQA web UI?

Reply:
Yes. Wanna to change it so something else?